src/should-forward-property.js

```diff
@@ -600,17 +26,18 @@ const isCustomAttribute = RegExp.prototype.test.bind(
 )
 
 const hasItem = (list, name) => list.indexOf(name) !== -1
-const isHtmlProp = name => hasItem(htmlProps, name)
+const isHtmlProp = (name, tagName) => {
+  const elementAttributes = reactHTMLAttributes[tagName]
+
+  return hasItem([...globalReactHtmlProps, ...elementAttributes], name)
```

Comment:
Actually the `tagName === 'svg'` lookup should be extremely fast. So I'd leave that where it is. Also you're probably right, putting the `isReactProp` before the `isCustomAttribute` would be faster. So we can swap those if you like. Hard to tell really without some hard benchmarks though so 🤷‍♂️